back/app/controllers/web_api/v1/insights/phase_insights_controller.rb

```diff
@@ -0,0 +1,31 @@
+module WebApi
+  module V1
+    module Insights
+      class PhaseInsightsController < ApplicationController
+        skip_before_action :authenticate_user
+        before_action :set_phase, only: %i[show_insights voting_insights]
+
+        def show_insights
```

Comment:
Might this be better as `index` - ie the default view. Might get round the odd looking '' in the routes?